extensions/crossbell/src/upload-file.tsx

```diff
@@ -0,0 +1,60 @@
+import { Clipboard, showToast, getSelectedFinderItems, showHUD, closeMainWindow } from "@raycast/api";
+import fs from "fs";
+import fsp from "fs/promises";
+import { ipfsUploadFile } from "crossbell/ipfs";
+import { fetch, FormData } from "node-fetch-native";
```

Comment:
I don't think we need this cuz node 18+ has `fetch` built-in.

Reply:
Tested on my local machine, there is no fetch and FormData, so I added node-fetch-native.